python-sdk/Makefile

```diff
@@ -1,6 +1,8 @@
 
 docs:
-	./docs/generate.py
+	docker build -t pachyderm_sdk_docs -f Dockerfile.docs-generation .
+	rm -rf docs/pachyderm_sdk
```

Comment:
What is this folder?

Reply:
Ah ok, this is where the docs are created and it removes it if it exists

Reply:
Yeah this is where the docs end up. I'll include this info the readme